src/jobs/npm_security_audit.yml

```diff
@@ -3,12 +3,15 @@ description: >
   Node.js projects using npm - auditing package dependencies for security vulnerabilities
 executor:
   name: node
-  tag: "16.15"
+  tag: <<parameters.node_version>>
```

Comment:
would prefer a `node_tag` parameter to match other usages e.g. https://github.com/ministryofjustice/hmpps-circleci-orb/blob/main/src/executors/node_redis.yml#L13